xarray/core/common.py

```diff
@@ -1010,8 +1010,10 @@ def _resample(
 
         if loffset is not None:
             emit_user_level_warning(
-                "Following pandas, the `loffset` parameter to resample will be deprecated "
-                "in a future version of xarray.  Switch to using time offset arithmetic.",
+                "Following pandas, the `loffset` parameter to resample "
+                "is deprecated in a future version of xarray.  "
```

Comment:
```suggestion
                "is deprecated.  "
```

Reply:
How about

```suggestion
                "is deprecated.  For example, 
                    >>> offset = pd.tseries.frequencies.to_offset(freq) / 2
    		    >>> resampled_ds["time"] = resampled_ds.get_index("time") + offset "
```

Seems like we should just support a `center=True`